### ISSUE TITLE
Keyboard navigation in autocomplete

### ISSUE THREAD

**Issue by matyunya** [matyunya/smelte#129]

**Comment by dennisjlee** [comment]
@matyunya I am using the Select in a project and I'd like to use autocomplete too, but the lack of keyboard navigation makes it a bit non-intuitive. I might be able to try to fix this in Smelte - are you open to pull requests to try to address this? Any prior thoughts about the right way to implement it?

**Comment by matyunya** [comment]
Sure, why not. There are many ways to implement it. Here's one I'm using in ellx.io. (Try Cmd+P on this page https://ellx.io/matyunya/cap-table/index.md)

https://gist.github.com/matyunya/41fc95eb7d196eef62909527525a01e6
